### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by mattheu**

### CODE REVIEW

**Comment by mattheu** [comment]
Either @sambulance or @peterwilsoncc - would be good to get a review on this.

File: src/html/components/buttons.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+<br/ >
+
+<a class="Btn Btn-Primary Btn-Small" href="#">Primary Button</a>
+<a class="Btn Btn-Secondary Btn-Small" href="#">Secondary Button</a>
+<a class="Btn Btn-Tertiary Btn-Small" href="#">Tertiary Button</a>
+
+<br/ >
+
+<a class="Btn Btn-Link" href="#">Plain button</a>
```
Comment (sambulance) [inline src/html/components/buttons.html]: Change this to a `<button>` element

File: src/html/components/buttons.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<h2>Buttons</h2>
+
+<a class="Btn Btn-Primary" href="#">Primary Button</a>
+<a class="Btn Btn-Secondary" href="#">Secondary Button</a>
+<a class="Btn Btn-Tertiary" href="#">Tertiary Button</a>
+
+<br/ >
```
Comment (sambulance) [inline src/html/components/buttons.html]: `<br />`

File: src/html/components/buttons.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+<input type="submit" class="Btn Btn-Primary" value="Primary Button" />
+<button class="Btn Btn-Secondary">Secondary Button</button>
+<a class="Btn Btn-Transparent" href="#">Plain button</a>
```
Comment (sambulance) [inline src/html/components/buttons.html]: Perhaps either make separate rows for each element (eg row of links, row of buttons, row of inputs) with a header above each one, or just keep this row which demonstrates mixed elements and add the transparent one here. Hope that makes sense?

File: src/html/components/buttons.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<h2>Buttons</h2>
+
+<a class="Btn Btn-Primary" href="#">Primary Button</a>
```
Comment (sambulance) [inline src/html/components/buttons.html]: There are no styles for `Btn-Primary`, so the class is redundant

File: src/html/index.html [hunk 5]
```diff
@@ -27,9 +27,11 @@ <h1 class="pattern-library-title">Juniper.</h1>
...
 	@include( './components/logos.html' )


 	@include( './components/code-block.html' )
+	@include( './components/buttons.html' )
+
 </div>
 
-<script src="/dist/assets/js/app.min.js"></script>
+<script src=."/assets/js/app.min.js"></script>
```
Comment (sambulance) [inline src/html/index.html]: The full stop should be inside the quotes

Comment (sambulance) [inline src/html/index.html]: The full stop should be inside the quotes

File: src/html/components/buttons.html (hunk 4 above)
Comment (sambulance) [inline src/html/components/buttons.html]: There are no styles for `Btn-Primary`, so the class is redundant

**Comment by mattheu** [comment]
@sambulance fixed!

File: src/styles/components/_Buttons.scss [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+	letter-spacing: 0;
+
+	&:link,
+	&:visited {
+		color: $hm-primary;
+		background: transparent;
+
+		&:hover,
+		&:focus {
+			text-decoration: underline;
```
Comment (sambulance) [inline src/styles/components/_Buttons.scss]: Already declared on line 107

File: src/styles/components/_Buttons.scss [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+	color: $hm-primary;
+
+	&:link,
+	&:visited {
+		color: $hm-primary;
+	}
+
+	&:hover,
+	&:focus {
+		color: white;
```
Comment (sambulance) [inline src/styles/components/_Buttons.scss]: Inconsistent `color` declaration - #FFF used elsewhere

File: src/styles/components/_Buttons.scss [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+	&:link,
+	&:visited {
+
+		color: #FFF;
+		text-decoration: none;
+
+		&:hover,
+		&:focus {
+			text-decoration: none;
```
Comment (sambulance) [inline src/styles/components/_Buttons.scss]: We can probably remove this `&:hover` block as text decoration is declared on line 34

File: src/styles/components/_Buttons.scss [hunk 9]
```diff
@@ -0,0 +1,116 @@
+.Btn {
+	@include font-heading;
+	@include text-std;
+	border: 1px solid $hm-primary;
```
Comment (sambulance) [inline src/styles/components/_Buttons.scss]: Border is the same as the background on all styles, perhaps we should set `border: 0;` and remove the colour declarations below?

File: src/html/components/buttons.html (hunk 3 above)
Comment (sambulance) [inline src/html/components/buttons.html]: @mattheu You should have an email with a comment I made here! Please can you paste in?

File: src/styles/components/_Buttons.scss (hunk 9 above)
Comment (mattheu) [inline src/styles/components/_Buttons.scss]: I think these can just be removed. I usually do this to try and make sure they are the same height as the form input fields, but this isn't required as we're using `* { box-sizing: border-box }`

**Pull request merged**